src/components/sendAskMoney/SendAskMoney.vue

```diff
@@ -91,47 +91,43 @@
           </div>
           <div class="container is-fluid custom-heavy-line-separator"></div>
         </div>
-        <section class="modal-card-body">
-          <div v-if="ownCurrenciesPartners.length !== 0">
-            <div  v-if="ownCurrenciesPartners">
-              <div class="container custom-width-send-money mt-4"
+        <section class="modal-card-body custom-modal-card-body">
+          
+          <div v-if="ownCurrenciesPartners.length !== 0" class="custom-card is-flex-direction-column is-align-items-center is-justify-content-space-between mb-4">
```

Comment:
Can you remove the bottom margin (the `mb-4` class) here please ? I find it useless.
There is already a huge top and bottom padding with the `modal-card-body` class on the parent.